tf2onnx/optimizer/transpose_optimizer.py

```diff
@@ -73,13 +73,12 @@ def pre_optimize_action(self):
                     for i, name in enumerate(child.input):
                         if name == output_name:
                             child.input[i] = const_name
-            ops.append(self._g.make_const(const_name, new_data))
+            self._g.make_const(const_name, new_data)
 
             # need call this to make input update synced to protobuf val
             self._g.update_proto()
-            ops.remove(reshape_op)
-            self._g.set_nodes(ops)
-            self._g.topological_sort(ops)
+            self._g.remove_node(reshape_op.name)
+            self._g.topological_sort(self._g.get_nodes())

```

Comment:
with this change, still need the ```ops``` as argument?
at least it can be optional.

Reply:
yeah, we might not need them now. let's do it separately next time.